Parsers/ASimAuthentication/Tests/GoogleWorkspace_ASimAuthentication_DataTest.csv

```diff
@@ -0,0 +1,11 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 3 records (0.1%) for field [EventResultDetails] of type [Enumerated]: [""MFA not satisfied""] (Schema:Authentication)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 3012 records (100.0%) for field [EventProduct] of type [Enumerated]: [""Workspace""] (Schema:Authentication)"
```

Comment:
@vakohl - Can you please fix these errors

Reply:
@anki-narravula I'm adding these new values in Tester.csv. I think these errors will be fixed once the new values are merged in tester.